jrportfolio.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Jason Rees Portfolio</title>	
+		<link href="normalize.css" rel="stylesheet" type="text/css">
```

Comment:
link tags are not inside the title, please indent them at the same level of title